CHANGELOG.md

```diff
@@ -10,6 +10,20 @@ All notable changes to [Earthly](https://github.com/earthly/earthly) will be doc
 ### Changed
 - `GIT CLONE` URLs will only be printed once as part of a prefix, e.g. `+my-clone-target(https://g/e/earthly) | --> GIT CLONE (--branch ) https://github.com/earthly/earthly`
 
+## v0.7.19 - 2023-09-20
```

Comment:
something looks wrong here -- why is the CHANGELOG being updated for a previously released version?

Reply:
not sure what happened here -- maybe a new commit was pushed, or some bug in github? CHANGELOG looks fine now.